src/PowerShellEditorServices/Services/DebugAdapter/Debugging/BreakpointApiUtils.cs

```diff
@@ -177,7 +177,7 @@ public static ScriptBlock GetBreakpointActionScriptBlock(string condition, strin
                 StringBuilder builder = new StringBuilder(
                     string.IsNullOrEmpty(logMessage)
                         ? "break"
-                        : $"Microsoft.PowerShell.Utility\\Write-Host '{logMessage}'");
+                        : $"Microsoft.PowerShell.Utility\\Write-Host \"{logMessage}\"");
```

Comment:
I didn't do this... but can you also escape any `"` inside logMessage?

Reply:
Sure. Do we have a utility method that does that?  Otherwise s/"/`"/ ?  I wonder if it's worth checking in the user already escaped the double quote?

Reply:
pretty sure we have one - let me check

Reply:
ahh we have only escaping of single quotes...

> s/"/`"/ 

This is probably fine for now... if someone complains we can fix it.

Reply:
Yeah, I can't imagine it would be a huge problem; you can already do anything inside a double-quoted string in PS

Reply:
Although it's fun to think of things like:

```
`"; Write-Host 'HA!'; #
```